packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -19,9 +19,42 @@ import {quote} from "shell-quote";
 import {BundleVariableModel} from "../bundle/models/BundleVariableModel";
 import {MsPythonExtensionWrapper} from "../language/MsPythonExtensionWrapper";
 import path from "path";
+import {isPowershell} from "../utils/shellUtils";
 
 const withLogContext = logging.withLogContext;
-const execFile = promisify(execFileCb);
+function getEscapedCommandAndAgrs(
+    cmd: string,
+    args: string[],
+    options: SpawnOptionsWithoutStdio
+) {
+    if (process.platform === "win32") {
+        args = [
+            "/d",
+            "/c",
+            `""${cmd}" ${args.map((a) => `"${a}"`).join(" ")}"`,
+        ];
+        cmd = "cmd.exe";
```

Comment:
Why are we changing the `cmd` here? In the `args` we will capture the old `cmd`, right? So it seems like this assignment should have any effects?

Reply:
Oh wait, we return it later on, makes sense

Reply:
We want to use cmd.exe to run this new command. Sort of like `bash -c "command"`